controllers/vmware/vspherecluster_reconciler.go

```diff
@@ -160,7 +161,7 @@ func (r *ClusterReconciler) reconcileDelete(clusterCtx *vmware.ClusterContext) {
 
```

Comment:
There should be some additions to the RBAC markers, right? (above Reconcile l.73)

Reply:
Thanks for catching this. I added it to controllers/vsphere_controllers.go along with other RBACK markers.